api/schemas/placeData.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
+description: More informations about a place
+# TODO decide which geocoding API we use and have this resource match its type
+properties:
+  address:
+    type: string
+    example: via Garibaldi, 1
+  city:
+    type: string
+    example: Padova
+  zipcode:
+    type: string
```

Comment:
non è meglio usare un integer?

Reply:
In effetti uno zipcode è un integer...

Reply:
In realtà, è giusto che sia una stringa: Alcuni paesi hanno zipcodes alfanumerici, o con leading zeros di format!
Oltretutto, non lo metterei obbligatorio, molti paesi non ce l'hanno

Reply:
No, vi faccio un esempio:
Castel Gandolfo, in provincia di Roma, il suo CAP è 00073. Se il tipo è number allora viene memorizzato come 73, ma ovviamente non va bene